test/unit/processing/statement/get-statement.test.js

```diff
@@ -0,0 +1,78 @@
+jest.mock('../../../../app/processing/organisation/get-organisation')
+const getOrganisation = require('../../../../app/processing/organisation/get-organisation')
+
+jest.mock('../../../../app/processing/invoice-line/get-invoice-line-by-funding-code-and-payment-id')
+const getInvoiceLineByFundingCodeAndPaymentId = require('../../../../app/processing/invoice-line/get-invoice-line-by-funding-code-and-payment-id')
+
+jest.mock('../../../../app/processing/calculation/get-calculation')
+const getCalculation = require('../../../../app/processing/calculation/get-calculation')
+
+jest.mock('../../../../app/processing/funding/get-fundings-by-calculation-id')
+const getFundingsByCalculationId = require('../../../../app/processing/funding/get-fundings-by-calculation-id')
+
+jest.mock('../../../../app/processing/payment-request/get-completed-payment-request-by-payment-request-id')
+const getCompletedPaymentRequestByPaymentRequestId = require('../../../../app/processing/payment-request/get-completed-payment-request-by-payment-request-id')
+
+jest.mock('../../../../app/processing/settlement/get-settled-settlement-by-settlement-id')
+const getSettledSettlementBySettlementId = require('../../../../app/processing/settlement/get-settled-settlement-by-settlement-id')
+
+const getStatement = require('../../../../app/processing/statement')
+
+let calculation
+let paymentRequest
+let settlement
+
+describe('get various components and transform to statement object', () => {
+  beforeEach(() => {
+    const retrievedOrganisationData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-organisation').rawOrganisationData))
+    const retrievedCalculationData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-calculation').rawCalculationData))
+    const retrievedFundingsData = JSON.parse(JSON.stringify(require('../../../mock-objects/mock-fundings').rawFundingsData))
+    const retrievedPaymentRequest = JSON.parse(JSON.stringify(require('../../../mock-payment-request').processingPaymentRequest))
+    const retreivedSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+
+    settlement = {
+      invoiceNumber: retreivedSettlement.invoiceNumber,
+      paymentRequestId: 1,
+      reference: retreivedSettlement.reference,
+      settled: retreivedSettlement.settled,
+      settlementDate: new Date(retreivedSettlement.settlementDate)
+
+    }

```

Comment:
Wrong way around: the mock objects are the 'complete' message whereas our 'retreived' equivalents are a subset/transformation of those